frontend-react/src/config/endpoints/settings.ts

```diff
@@ -5,6 +5,7 @@ export enum ServicesUrls {
     SENDERS = "/settings/organizations/:orgName/senders",
     SENDER_DETAIL = "/settings/organizations/:orgName/senders/:sender",
     RECEIVERS = "/settings/organizations/:orgName/receivers",
+    PUBLIC_KEYS = "/settings/organizations/:organizationName/public-keys",
```

Comment:
we should stick to the :orgName pattern

Reply:
Agree, unfortunately the backend API is `organizationName`. @stephenkao should I update to `orgName` in backend to stay consistent throughout?

Reply:
The segments here like `:organizationName` are arbitrary names so we could change this to `:orgName` (or anything that's a valid variable name) and just make sure we pass in an `orgName` in `segments` [here](https://github.com/CDCgov/prime-reportstream/pull/9007/files/07620cb92bd7e08fed5d6f67199eca3b45bbb160#diff-624d4d47c219d2c816d15fb78b90941179412aa9149d8a5a7ce637195a856c8cR21):

```ts
segments: {
    orgName: activeMembership?.parsedName!!,
}
```